mitodl/ocw-hugo-themes
# fix: accessibility fixes

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/385

#### What's this PR do?

- [x] typo in https://github.com/mitodl/ocw-hugo-themes/blob/eabcc2e8b0e71010cd8e38bbac41d52d0d209582/www/layouts/educator/section.html#L433 -- should be "And what does it mean to truly listen, to truly understand?
- [x] link "Chalk Radio" to https://chalk-radio.simplecast.com/ here https://github.com/mitodl/ocw-hugo-themes/blob/eabcc2e8b0e71010cd8e38bbac41d52d0d209582/www/layouts/educator/section.html#L274-L275 and here https://github.com/mitodl/ocw-hugo-themes/blob/eabcc2e8b0e71010cd8e38bbac41d52d0d209582/www/layouts/educator/section.html#L346
- [x] remove this link, because we won't implement it soon: https://github.com/mitodl/ocw-hugo-themes/blob/eabcc2e8b0e71010cd8e38bbac41d52d0d209582/www/layouts/educator/section.html#L317
- [x] in https://github.com/mitodl/ocw-hugo-themes/blob/eabcc2e8b0e71010cd8e38bbac41d52d0d209582/www/layouts/educator/section.html#L306 change the link to "coming soon" until we add a "Instructor Insights" facet, in #384 
- [x] Change these both to "Explore How We Teach at MIT" https://github.com/mitodl/ocw-hugo-themes/blob/eabcc2e8b0e71010cd8e38bbac41d52d0d209582/www/layouts/educator/section.html#L288 and https://github.com/mitodl/ocw-hugo-themes/blob/eabcc2e8b0e71010cd8e38bbac41d52d0d209582/www/layouts/educator/section.html#L63
- [x] typo "Adaptation" instead of "Adaption" throughout the page, including file names and ids
- [x] typo "departments" instead of "departmen" 
- [x] typo and formatting `&ldquo;Thinking like an Economist&rdquo;` instead of `Thinkin Like an &ldquo;Economist&rdquo;`
- [x] accessibility: JavaScript is blocking the normal navigation of anchor links in screen readers. 
       Since screen readers already have tools for navigating the heading structure, I think it would be best to just hide the top navigation from the screenreader. 
- [x] All decorative images, which is _all_ the images on this page, should have `alt=""` as they don't do anything to aid the screen reader user's understanding of the content and are just an annoying distraction. 
- [x] slight change to the checkbox for "Change these both to "Explore How We Teach at MIT""

#### How should this be manually tested?
Go to [educator page](https://ocw-hugo-themes-pr-389--ocw-next.netlify.app/educator/) and verify all the things mentioned above in `What's this PR do?`

#### Screenshots (if appropriate)
To be provided
